### PR TITLE
fix: migration with proper multisig

### PR DESCRIPTION
**Pull request by antmat**
subj, now we use the proper multisig contract

### CODE REVIEW

File: blockchain/source/migrations/2_deploy_v1.js [hunk 1]
```diff
@@ -134,21 +135,25 @@ async function deploySidechain (deployer, network, accounts) {
...
+
     await ahm.write('blacklistAddress', bl.address, { gasPrice: 0 });
     await ahm.write('marketAddress', market.address, { gasPrice: 0 });
     await ahm.write('profileRegistryAddress', pr.address, { gasPrice: 0 });
     await ahm.write('oracleUsdAddress', oracle.address, { gasPrice: 0 });
     await ahm.write('gatekeeperSidechainAddress', gk.address, { gasPrice: 0 });
     await ahm.write('multiSigAddress', multiSig.address, { gasPrice: 0 });
     // compatibility
-    await ahm.write('migrationMultSigAddress', multiSig.address, { gasPrice: 0 });
+    await ahm.write('migrationMultiSigAddress', multiSig.address, { gasPrice: 0 });
```
Comment (sokel) [inline blockchain/source/migrations/2_deploy_v1.js]: you're write the same address for `migrationMultiSigAddress` and `multiSigAddress` ?

Comment (antmat) [reply blockchain/source/migrations/2_deploy_v1.js]: yep, that's right

Comment (antmat) [reply blockchain/source/migrations/2_deploy_v1.js]: ok, maybe it is not a good idea, if we still will have several multisigs in future

Comment (sokel) [reply blockchain/source/migrations/2_deploy_v1.js]: ok, it's really right.

**Pull request merged**